website/docs/api-loss.md

```diff
@@ -81,22 +91,29 @@ normalize = true
 
 </grid>
 
-| Argument       | Type          |  Description                                      |
-| -------------- | ------------- | ------------------------------------------------- |
-| _keyword-only_ |               |                                                   |
-| `normalize`    | <tt>bool</tt> | Normalize and divide by number of examples given. |
+| Argument          | Type                     |  Description                                                        |
+| ----------------- | ------------------------ | ------------------------------------------------------------------- |
+| _keyword-only_    |                          |                                                                     |
+| `names`           | <tt>List[str]</tt>       | Label names. Has to be provided when using with List[str] as truths. |
+| `normalize`       | <tt>bool</tt>            | Normalize and divide by number of examples given.                   |
+| `neg_prefix`      | <tt>str</tt>             | Symbol that indicates that a label is negative e.g. "!sci-fi".      |
+| `missing_value`   | <tt>Union[str, int]</tt> | Symbol for "missing value" among the labels.                        |
```

Comment:
Alternatively: "Specific label that indicates the value is missing and should not be considered for training/evaluation purposes, e.g. empty string `""` or `0`." ?